crates/burn-tensor/src/tests/ops/any.rs

```diff
@@ -0,0 +1,46 @@
+#[burn_tensor_testgen::testgen(any)]
+mod tests {
+    use super::*;
+    use burn_tensor::{Data, Tensor};
+
+    #[test]
+    fn test_any() {
```

Comment:
I think we should also have versions of this test that give false. Otherwise the op could just be `return true`